src/components/Chat.tsx

```diff
@@ -310,9 +313,9 @@ const Chat: React.FC<ChatProps> = ({
   };
 
   useEffect(() => {
-    async function handleSelection(selectedText: string) {
-      const wordCount = selectedText.split(" ").length;
-      const tokenCount = await chainManager.chatModelManager.countTokens(selectedText);
+    async function handleSelection(event: CustomEvent) {
```

Comment:
Nice! As a note: we just need to make sure text selection commands/prompts are working

Reply:
I tested it in this way:
https://www.dropbox.com/scl/fi/vgpcjlwig34xzz3t96wz7/custom-prompt.mov?rlkey=f4gfslsf8vx5wf2i4uczub5dw&st=o1ju6xno&dl=0